rust.md

```diff
@@ -26,4 +26,15 @@ export RUSTFLAGS="-Ctarget-cpu=neoverse-n1"
 cargo build --release
 ```
 
+If the Rust compiler uses LLVM 12 or later, you can use `export
+RUSTFLAGS="-Ctarget-feature=+outline-atomics"` to produce a binary that runs on
+both Graviton and Graviton2.  LLVM 12 added support for `outline-atomics` which
```

Comment:
@sebpop do we need this for graviton2 only customers?   

ARMv8.0 is so rare in the market these days that i dont think its worth the clutter

Reply:
This target feature is only for customers who need compatibility with old Armv8.0 hardware.  @AGSaidi do you agree to abandon this patch?

Reply:
I think we want rust code to do this by default so that if someone distributes a rust app they get good performance.